study1/week2/invert-binary-tree.java

```diff
@@ -0,0 +1,33 @@
+//https://leetcode.com/problems/invert-binary-tree/description/
+
+/**
+ * Definition for a binary tree node.
+ * public class TreeNode {
+ *     int val;
+ *     TreeNode left;
+ *     TreeNode right;
+ *     TreeNode() {}
+ *     TreeNode(int val) { this.val = val; }
+ *     TreeNode(int val, TreeNode left, TreeNode right) {
+ *         this.val = val;
+ *         this.left = left;
+ *         this.right = right;
+ *     }
+ * }
+ */
+
+class Solution {
+    public TreeNode invertTree(TreeNode root) {
+        //time complexity: O(nlogn) bacause it is binary Tree archtecture 
```

Comment:
Can you try to think again? how many nodes are you visiting?

Reply:
n nodes....? then is O(n) right?